src/streamer/processors/asyncOpStream.ts

```diff
@@ -33,18 +41,19 @@ export default class AsyncOpStreaming extends SObjectStreaming {
   protected asyncOpStreamProcessor(payload: JsonMap): StatusResult {
     const jsonPayload = ensureJsonMap(payload);
 
-    // Print the message from the payload if it exists
-    if (jsonPayload.sf_devops__Message__c) {
-      // eslint-disable-next-line no-console
-      console.log(jsonPayload.sf_devops__Message__c);
-    }
+    // We build an aor given the payload
+    const asyncOpResult: AsyncOperationResult = {
+      Id: jsonPayload.Id as string,
+      sf_devops__Error_Details__c: jsonPayload.sf_devops__Error_Details__c as string,
+      sf_devops__Status__c: jsonPayload.sf_devops__Status__c as AsyncOperationStatus,
+      sf_devops__Message__c: jsonPayload.sf_devops__Message__c as string,
```

Comment:
Can't you just cast jsonPayload to a AsyncOperationResult?   They have the same members.

Reply:
:ok: